ftw/footer/tests/test_footer_functional.py

```diff
@@ -19,16 +18,35 @@ def setUp(self):
 
     @browsing
     def test_footer_on_non_contentish_object(self, browser):
-        # This tests checks tests if the footer is rendered on non contentish
-        # objects. For instance the view on an address portlet.
-        browser.login().open(
-            self.portal,
-            {':action': "/++contextportlets++ftw.footer.column1/"
-                        "+/plone.portlet.static.Static"})
-        browser.fill({'Text The text to render': 'Blubber'}).submit()
-
-        browser.open('http://nohost/plone/++contextportlets++'
-                     'ftw.footer.column1/portlet_static/edit')
+        """
+        The footer must be rendered on non-contentish views too, e.g.
+        the Plone control panel.
+        """
+        browser.login()
+
+        # Add a footer portlet.
+        browser.open(self.portal.absolute_url() + '/++contextportlets++ftw.footer.column1/+/plone.portlet.static.Static')
+        browser.fill({
+            'Text The text to render': 'My Portlet Text',
+        }).save()
+
+        # The footer portlet is shown on the Plone Site.
+        browser.open(self.portal)
+        self.assertEqual(
+            'My Portlet Text',
+            browser.css('#footer-column-1').first.text
+        )
+
+        # The footer portlet is shown on the control panel.
+        browser.open(self.portal.absolute_url() + '/@@overview-controlpanel')
```

Comment:
This is no longer the same test. The control panel is on a contentish object (Plone Root)
But it should also work, for example on a portlet context: `http://nohost/plone/++contextportlets++ftw.footer.column1/portlet_static/edit`